AnnService/inc/Core/Common.h

```diff
@@ -81,6 +81,15 @@ inline bool fileexists(const TCHAR* path) {
     return (dwAttr != INVALID_FILE_ATTRIBUTES) && (dwAttr & FILE_ATTRIBUTE_DIRECTORY) == 0;
 }
 #define mkdir(a) CreateDirectory(a, NULL)
+
+#ifndef max
+#define max(a,b) (((a) > (b)) ? (a) : (b))
```

Comment:
I think we should prefer to use `std::max`, define this macro can cause some conflicts with other code

Reply:
this is required by dll compiling. It seems it cannot find the std::max.